src/components/modals/HintModal.tsx

```diff
@@ -0,0 +1,36 @@
+//import EndQuote from '../../img/EndQuote.png'
+import openQuote from '../../img/OpenQuote.png'
+import { BaseModal } from './BaseModal'
+
+type Props = {
+  isOpen: boolean
+  handleClose: () => void
+  verseText: string
+}
+
+export const HintModal = ({ isOpen, handleClose, verseText }: Props) => {
+  return (
+    <BaseModal title="Hint" isOpen={isOpen} handleClose={handleClose}>
+      <img
+        src={openQuote}
+        alt="OpenQuote.png"
```

Comment:
Let's give a different alt value than the name of the file. Perhaps `begin quote`?

Reply:
How about this?

Reply:
@kenancasey ??

Reply:
Actually since it's not an image that conveys content, isn't an emtpy alt recommended?

Reply:
https://www.w3.org/WAI/tutorials/images/decorative/

Reply:
I'm good with that, but react might give you a warning. Either way, I don't like the file name.